applitools.config.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   serverUrl: 'https://twilioeyesapi.applitools.com',
-  batchId: process.env.GITHUB_HEAD_SHA,
```

Comment:
Removed these to allow me to run the eyes checks locally. This is essentially replaced with a bit more involved logic checking to see if the cypress run is in the context of a local run, or a CI run (e.g. via github workflows)